cmd/cmd.go

```diff
@@ -42,11 +42,12 @@ const alignSize = 4096
 
 // flags
 var (
-	serial    = false
-	verbose   = false
-	blockSize = "4MiB"
-	fileSize  = "1GiB"
-	cpuNode   = 0
+	serial     = false
+	verbose    = false
+	blockSize  = "4MiB"
+	fileSize   = "1GiB"
+	cpuNode    = 0
+	ioPerDrive = 2
```

Comment:
```suggestion
	ioPerDrive = 4
```

Let's keep the default as 4 since it's the optimal number of concurrency that we atleast need to see good numbers with. When a drive involves in MinIO it does perform more than 4 threads per I/O.

Reply:
Keep ioPerDrive as 4